Applications/DataExplorer/VtkVis/VtkVisTabWidget.cpp

```diff
@@ -326,16 +326,15 @@ void VtkVisTabWidget::buildProportiesDialog(VtkVisPipelineItem* item)
             QString key = i.key();
             QList<QVariant> values = i.value();
 
-            VtkAlgorithmPropertyVectorEdit* vectorEdit;
             if (!values.empty())
             {
                 QList<QString> valuesAsString;
                 foreach (QVariant variant, values)
                 valuesAsString.push_back(variant.toString());
 
-                vectorEdit = new VtkAlgorithmPropertyVectorEdit(valuesAsString, key,
-                                                                values.front().type(),
-                                                                algProps);
+                VtkAlgorithmPropertyVectorEdit* vectorEdit =
```

Comment:
Have you tested this? Does the UI still find the array?

Reply:
Is this (and actually the same question above) still open, or can we merge?

Reply:
@endJunction I haven't had any feedback regarding the PropertyVector.

Tom tested the QAction-thing and it seems to be working fine.

Reply:
@rinkk What are the necessary steps to test if the changes are okay?

Reply:
Apply any filter with a multi-value parameter to a dataset to see if the widgets are displayed correctly.

E.g. apply a threshold filter to a mesh and check if two line edits are displayed for the "Range"-parameter.

Reply:
Tested this, threshold filter works. So my changes don't destroy anything in principle.